### ISSUE TITLE
PCC connector for .NET core application via SteelToe

### ISSUE THREAD

**Issue by VenkateshSrini** [SteeltoeOSS/steeltoe#69]
Hi,
I recently saw an video where a .NET program was able to connect to PCC using Pivotal.GemFire library. Can you please let us if this connectivity will be available as part of SteelToe Connector .NET standard library next release. The problem is I do not see a Pivotal.GemFire .NET standard version.

**Comment by TimHess** [comment]
We have an initial implementation (see the `gemfire_connector` branch if you'd like), but until there's a NuGet distribution of the driver package (or a workaround for our CI), our unit tests won't compile or pass. 

Samples for ASP.NET and ASP.NET Core are in the [gemfire branch of the samples repo](https://github.com/SteeltoeOSS/Samples/tree/gemfire)

**Comment by VenkateshSrini** [comment]
Hey,
I see this lines
<PropertyGroup>

    <TargetFramework>net461</TargetFramework>

    <DebugType>full</DebugType>

    <Platforms>x64</Platforms>

  </PropertyGroup>

This means that even the asp.net core is also going to run only on the .NET framework? I would expect the TragetFramework to netcoreapp2.0 or 3.0 so that it is cross platform

**Comment by TimHess** [comment]
Unfortunately the GemFire driver is currently built exclusively for 64 bit Windows. I actually did the work to support it in Steeltoe back in February (the dates in source history are just when I moved the code into the mono-repo), a few months before netcore 3.0 was announced.
We are likely to revisit the targeting of a few Steeltoe packages for our 3.0 release

**Comment by TimHess** [comment]
This connector is now available in 2.3.0-rc2, please try it out!

Samples are [here ](https://github.com/SteeltoeOSS/Samples/tree/2.x/Connectors/src/AspDotNet4)and [here ](https://github.com/SteeltoeOSS/Samples/tree/2.x/Connectors/src/AspDotNetCore/GemFire)

**Comment by VenkateshSrini** [comment]
Hi,
I see the below code
```
<ItemGroup>

    <Reference Include="Pivotal.GemFire">

      <HintPath>..\..\..\Pivotal.GemFire.dll</HintPath>

    </Reference>

  </ItemGroup>
```

This means that it is added from local path. How would I get this Pivotal.GemFire.dll

**Comment by TimHess** [comment]
Sorry, forgot to link to the [docs for the GemFire connector](https://steeltoe.io/docs/steeltoe-connectors/#8-1-usage). 

The Pivotal GemFire client is distributed by Pivotal on https://network.pivotal.io/ and the latest version is [10.0.2](https://network.pivotal.io/products/pivotal-gemfire/#/releases/396680)

**Comment by VenkateshSrini** [comment]
There is a warning that says GemFire client runs only Windows 64 bit. I'm asking for cross platform .NET core, will that be available as part of 2.3.0 GA

**Comment by TimHess** [comment]
Sorry, that is a limitation of the underlying Geode/GemFire driver that has nothing to do with Steeltoe. The Steeltoe pieces of this puzzle are all cross platform already and would work today if the GemFire client supported it.
